### ISSUE TITLE
"HexCode" in colors_all

### ISSUE THREAD

**Issue by ekiefl** [Arcadia-Science/arcadia-pycolor#40]
```python
import arcadia_pycolor as apc
assert "HexCode" in apc.colors_all
```

A quick fix, given the current import structure, would be (untested):

```python
colors_all = [name for name in dir(colors) if isinstance(getattr(colors, name), HexCode)]
```

**Comment by mezarque** [comment]
Nice, this seems to work! Thanks for noticing that!

**Comment by mezarque** [comment]
Resolved by #46
